### PR TITLE
feat: Create Reset Password component for web ui

### LINKED ISSUE #35: [FEATURE] create Reset Password component for web ui
**Is your feature request related to a problem? Please describe.**
Create the component for the forgot password page component

**Describe the solution you'd like**
Should use material design consistent with the dev spec
Should follow the design of the created mockup
Should be a stateless component that only displays the form

**Additional context**
Branch: ft-reset-pasword-web-ui

### PR DESCRIPTION
**Pull request by Faithaigo**
## Description
Created reset password component for web ui
-Created a stateless component that displays only the from
-Used Material UI design

Closes #35

## How Has This Been Tested?
Unit test file included, test passing

### CODE REVIEW

File: src/web_ui/src/components/Forms/ResetPassword.js [hunk 1]
```diff
@@ -1,8 +1,81 @@
...
+        <StyledCard>
+            <CardContent>
+                <Div>
+                    <Img src="/images/resetimg.jpg"></Img>
+                </Div>
+                <Wrapper>
+                    <H1>Farmrail</H1>
+                    <H6>RESET PASSWORD</H6>
+                    <form>
+                        <Input type="text"/>
```
Comment (amkayondo) [inline src/web_ui/src/components/Forms/ResetPassword.js:72]: add a placeholder for the input

File: src/web_ui/src/components/Forms/ResetPassword.js [hunk 2]
```diff
@@ -1,8 +1,81 @@
...
+`
+
 
 const ResetPassword = () =>{
     return (
-        <div/>
+        <StyledCard>
+            <CardContent>
+                <Div>
+                    <Img src="/images/resetimg.jpg"></Img>
```
Comment (amkayondo) [inline src/web_ui/src/components/Forms/ResetPassword.js:66]: add an alternative text placeholder for the image 
also, place the image in an images folder like **src/assests/images/auth** for easy access by other components

Comment (amkayondo) [reply src/web_ui/src/components/Forms/ResetPassword.js:66]: Also, it's a good practice to use **``** than **`</Img>`**  if the element does not surround any content like **<div>{ component }</div>**

**Review by amkayondo — CHANGES_REQUESTED** [review]
Add tests for this component and also fix the needed changes

**Review by kmwamasali — APPROVED** [review]
Changes approved

**Pull request merged**